sound/soc/sof/sof-priv.h

```diff
@@ -620,10 +646,20 @@ void snd_sof_get_status(struct snd_sof_dev *sdev, u32 panic_code,
 int snd_sof_init_trace_ipc(struct snd_sof_dev *sdev);
 void snd_sof_handle_fw_exception(struct snd_sof_dev *sdev);
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_VIRTIO_FE)
+int sof_vfe_pcm_copy_user(struct snd_soc_component *component,
+			  struct snd_pcm_substream *substream, int channel,
+			  unsigned long pos, void __user *buf,
+			  unsigned long bytes);
+#else
+#define sof_vfe_pcm_copy_user NULL
+#endif
```

Comment:
Shouldn't this be static inline {} since this may be called by others SOF users directly ?

Reply:
@lgirdwood no, it isn't called by its name, it wouldn't compile in such a case. It's used to initialise a .ops method.